app/client/cli/cmd.go

```diff
@@ -2,38 +2,46 @@ package cli
 
 import (
 	"context"
-	"os"
 
-	"github.com/pokt-network/pocket/logger"
+	"github.com/pokt-network/pocket/runtime/configs"
 	"github.com/pokt-network/pocket/runtime/defaults"
 	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
 )
 
 const (
 	cliExecutableName = "client"
-	keybaseSuffix     = "/keys"
 )
 
 var (
 	remoteCLIURL   string
 	dataDir        string
+	configPath     string
 	nonInteractive bool
+	cfg            configs.Config
```

Comment:
Good catch, I made it a pointer to keep the same return type as ParseConfig.

Not sure about it ever being null though, I guess that would require refactoring more of the runtime since I think it everywhere just assumes it's not null.